typescript/phoenix-sdk/examples/decodeMainnetPhoenixTransaction.ts

```diff
@@ -0,0 +1,30 @@
+import { Connection } from "@solana/web3.js";
+
+import * as Phoenix from "../src";
+
+export async function decodeTransaction() {
```

Comment:
please add a comment for what this does

might be nice to just print out the expected output

Reply:
sure, can just embed the expected JSON output in the comment